gtsfm/utils/plane.py

```diff
@@ -0,0 +1,41 @@
+"""Utilities for generating data on planar surfaces.
+
+Authors: Ayush Baid
+"""
+from typing import Tuple
+
+import numpy as np
+
+
+def sample_points_on_plane(
+    plane_coefficients: Tuple[float, float, float, float],
+    range_x_coordinate: Tuple[float, float],
+    range_y_coordinate: Tuple[float, float],
```

Comment:
range_x and range_y are shorter names without much loss of readability.

Reply:
good point, i'll update that

Reply:
updated now